packages/nextjs/src/server/utils.ts

```diff
@@ -97,7 +97,19 @@ export function decorateRequest(
   req: ClerkRequest,
   res: Response,
   requestState: RequestState,
-  requestData?: AuthenticateRequestOptions,
+  requestData: AuthenticateRequestOptions,
+  publicRequestDate: Pick<
```

Comment:
Can you remind me why we need these two different buckets of request data?

Reply:
turns out i don't need the rest of the config, i only need a way for middleware to notify the ClerkProvider if the keys used are coming from the accountless cookies to avoid syncing constantly.

Reply:
https://github.com/clerk/javascript/pull/4602#discussion_r1860455702